judge/views/contests.py

```diff
@@ -192,6 +194,11 @@ def get_context_data(self, **kwargs):
         context['is_spectator'] = self.is_spectator
         context['can_edit'] = self.can_edit
 
+        participation_type = self.object.get_join_type(self.request.user)
+        context['can_test'] = participation_type == ContestParticipation.TESTING
```

Comment:
Does `TESTING` exist? Also, `can_test` is not currently used in any templates.
```py
>>> from judge.models import ContestParticipation
>>> ContestParticipation.LIVE
0
>>> ContestParticipation.TESTING
Traceback (most recent call last):
  File "<console>", line 1, in <module>
AttributeError: type object 'ContestParticipation' has no attribute 'TESTING'
```

Reply:
Yeah this is a mixup I made with #1941.